code/cgame/cg_credits.cpp

```diff
@@ -442,7 +437,7 @@ void CG_Credits_Init( const char *psStripReference, vec4_t *pv4Color)
 					{
 						// sort entries RHS dotted entries by alpha...
 						//
-						qsort(&CreditLine.vstrText[0], CreditLine.vstrText.size(), sizeof(CreditLine.vstrText[0]), SortBySurname);
+						std::sort( CreditLine.vstrText.begin(), CreditLine.vstrText.end(), SortBySurname );
```

Comment:
Compared to the previous code, this will additionally require the header <algorithm>.
It seems headers are mostly not included in this implementation file (i.e. <string> or <vector> isn't included directly). It compiled on my machine, so I left the includes as-is. Let me know if I need to add that somewhere to accommodate this change.